src/app/security/maturity-model/components/TableOfContents.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { clsx } from 'clsx'
+
+import { coreFunctionsData } from '../data/coreFunctionsData'
+import { scrollToSection } from '../utils/scrollToSection'
+import { useUrlHash, HASH_SIGN, type SectionHash } from '../utils/useUrlHash'
+
+export function TableOfContents() {
+  const { currentHash } = useUrlHash()
+
+  return (
+    <table role="navigation" aria-label="Table of Contents">
+      <p className="mb-4 ml-6 text-sm font-semibold uppercase text-brand-200 lg:mb-6">
+        Table of Contents
+      </p>
```

Comment:
<img width="420" alt="Screenshot 2024-09-12 at 10 13 02" src="https://github.com/user-attachments/assets/b595ceb1-3010-4677-b13a-2749c5ce257e">

Reply:
Good find! I was trying to use more explicit markup, as per your article recommendation, but it didn't work. I'll stick to `nav` for now.